middle_end/flambda2/to_cmm/to_cmm_set_of_closures.ml

```diff
@@ -105,7 +105,7 @@ end) : sig
 end = struct
   (* The [offset]s here are measured in units of words. *)
   let fill_slot for_static_sets decls dbg ~startenv value_slots env res acc
-      ~slot_offset updates slot =
+      ~slot_offset updates slot ~is_last =
```

Comment:
I think it would be cleaner if `is_last` was a field of `Slot_offsets.Function_slot`, rather than recomputing the layout in `fill_layout0` just to determine this parameter.

Reply:
I've asked @Gbury to opine on this.

Reply:
That's a good point. I can make a patch to correctly generate that when we compute the layout in `slot_offsets.ml`.

Reply:
This has been merged into this PR.  I have reviewed the change.